### PR TITLE
Switch to bsd license

### PR DESCRIPTION
**Pull request by blakemurdock**
## Please make sure you provide the information below:
Converting to the BSD License from the CC license

### Short description of changes:
This PR added the license and updated all dist/ level minified files to reference the new license. 

### Breaking in the external API:
N/A

### Breaking changes in the internal API:
N/A

### Todos/Notes:
Need to update wavesurfer-js.org to reflect these changes

### Related Issues and other PRs:
#1050

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) 2017, katspaugh
```
Comment (thijstriemstra) [inline LICENSE]: I'd say katspaugh and contributors. And look up when first commit of project was done and use that in copyright date, e.g. 2014-2017

File: package.json [hunk 2]
```diff
@@ -15,7 +15,7 @@
     "url": "git://github.com/katspaugh/wavesurfer.js.git"
   },
   "author": "",
-  "license": "CC-BY-3.0",
+  "license": "BSD 3-Clause License",
```
Comment (thijstriemstra) [inline package.json]: incorrect identifier, it should be `BSD-3-Clause`.

**Comment by blakemurdock** [comment]
@thijstriemstra I am seeing the miss on the bower.json. Where else is needed for the README? I changed the license mentions there.

**Comment by thijstriemstra** [comment]
https://github.com/katspaugh/wavesurfer.js#license

I suggest you do a grep for 'Creative commons' through all files, probably are more.

**Comment by blakemurdock** [comment]
That's the original README from the katspaugh repo. The change is in my fork:
https://github.com/blakemurdock/wavesurfer.js/blob/d4868431acfc60c9dfb3e148b60bb2e5ef59e1aa/README.md

I'll run the grep while fixing the other changes to fully verify I haven't missed anything else.

File: example/audio-element/index.html [hunk 3]
```diff
@@ -99,11 +99,11 @@ <h1 itemprop="name">Media Element Fallback Example</h1>
 
             <div class="footer row">
                 <div class="col-sm-12">
-                    <a rel="license" href="https://creativecommons.org/licenses/by/3.0/deed.en_US"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/3.0/80x15.png" /></a>
+                    <a rel="license" href="https://opensource.org/licenses/BSD-3-Clause"><img alt="BSD--Clause License" style="border-width:0" src="https://img.shields.io/badge/License-BSD%203--Clause-blue.svg" /></a>
```
Comment (thijstriemstra) [inline example/audio-element/index.html]: the `3` is missing in the img `alt` for all examples.

**Comment by katspaugh** [comment]
Thanks much, @blakemurdock and @thijstriemstra!

**Comment by thijstriemstra** [comment]
@katspaugh lets release it as 1.4.0?

**Comment by katspaugh** [comment]
Released 👍

**Pull request merged**